src/data/resolvers/colonyActions.ts

```diff
@@ -149,16 +161,28 @@ export const colonyActionsResolvers = ({
            */
           .filter((log) => !!log) as ProcessedEvent[];
 
-        let actionType = getActionType(reverseSortedEvents);
+        let actionType;
+        const motionCreatedEvent = reverseSortedEvents.find(
+          ({ name }) => name === ColonyAndExtensionsEvents.MotionCreated,
+        );
 
+        if (motionCreatedEvent) {
+          actionType = await getMotionActionType(
+            votingClient as ExtensionClient,
+            colonyClient as ColonyClient,
+            reverseSortedEvents[0],
+          );
+        } else {
+          actionType = getActionType(reverseSortedEvents);
+        }
         const actionValues = await getActionValues(
           reverseSortedEvents,
           colonyClient as ColonyClient,
+          votingClient as ExtensionClient,
```

Comment:
This doesn't work when the `VotingReputation` extension is not installed. It will break loading a normal action, since the voting client won't be available